openpower/package/petitboot/petitboot.mk

```diff
@@ -19,6 +19,8 @@ PETITBOOT_CONF_OPTS += --with-ncurses --without-twin-x11 --without-twin-fbdev \
 	      HOST_PROG_SHUTDOWN=/usr/libexec/petitboot/bb-kexec-reboot \
 	      $(if $(BR2_PACKAGE_BUSYBOX),--with-tftp=busybox)
 
+PETITBOOT_AUTORECONF_ENV += PETITBOOT_VERSION=`cat $(PETITBOOT_VERSION_FILE) | cut -d '-' -f 2-`
```

Comment:
I'm still  confused how `PETITBOOT_AUTORECONF_ENV` is needed and it's potential use?

the skiboot reference and others like https://github.com/open-power/op-build/blob/master/openpower/package/hostboot/hostboot.mk#L19 just overwrite the same variable.